app/styles/app.css

```diff
@@ -230,24 +230,37 @@ nav a:hover {
     justify-content: center;
 }
 
-.SearchableMetricSelector .modal-content {
+.searchableMetricSelector .modalContent {
     width: 500px;
     background-color: white;
 }
 
-.SearchableMetricSelector .modal-body {
+.searchableMetricSelector .modalBody {
     border-top: 1px solid #eee;
     border-bottom: 1px solid #eee;
     max-height: calc(100vh - 210px);
     overflow-y: auto;
 }
 
-.SearchableMetricSelector .modal-header, 
-.SearchableMetricSelector .modal-footer {
+.searchableMetricSelector .searchBar {
+    border-bottom: 1px solid #eee;
+}
+
+.searchableMetricSelector .searchResults p {
+    border-bottom: 1px solid #D3D3D3;
+}
+
+.searchableMetricSelector .searchResults p:hover {
+    background-color: rgba(180, 180, 180, 0.5);
+    cursor: pointer;
```

Comment:
Love it, cursor: pointer is a great addition to let the user know something is clickable.